app/views/user_mailer/welcome_email.html.haml

```diff
@@ -9,8 +9,12 @@
 
 %p
 
-  The 2022 summer by-elections for Wakefield, and Tiverton & Honiton
-  are on <em>Thursday, 23rd June</em>.
+  - if general_election?
+    The #{election_date_season_type} is
+  - else
+    The #{election_date_season_type} for #{by_election_constituencies_as_sentence} are
+
+  on <em>Thursday, #{election_date_dm}</em>.
```

Comment:
It might not be a Thursday ;-)

Reply:
Seems pretty unlikely not to be though :). "Since 1918 a General Election has always been on a Thursday, except for 1918, 1922, 1924 and 1931. ". In the rare event that it is not, we can either automate the change or do it manually . 

https://www.theguardian.com/notesandqueries/query/0,5753,-2036,00.html

Reply:
Oh interesting, I didn't know that!  Is that also true for by elections too?

Reply:
Follow-up in #686.